src/searchbar.cpp

```diff
@@ -80,6 +80,14 @@ SearchBarLineEdit::SearchBarLineEdit(QWidget *parent) :
     m_suggestionView->setRootIsDecorated(false);
     m_suggestionView->setStyleSheet(KiwixApp::instance()->parseStyleFromFile(":/css/popup.css"));
 
+    /* See line-height&padding resources/css/popup.css QHeaderView::section. */
+    m_suggestionView->setIconSize(QSize(24, 24));
```

Comment:
Does Qt provide a way of obtaining values from the stylesheet in effect?

Note that this comment applies to other magic number (CSS property values duplicated/hardcoded in C++ code) of this commit.

Reply:
I don't think its possible unless we parse the entire CSS to a global object like a JSON object. That will be a whole new project in itself.